### PR TITLE
add new select action

### PR DESCRIPTION
**Pull request by shulkaolka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,9 +202,14 @@ Following types are supported:
  * ``float``
  * ``date``
 
-![image](https://user-images.githubusercontent.com/40201204/43644974-332f2aa4-9739-11e8-8483-f7395e5d195d.png)
+![image](https://user-images.githubusercontent.com/16806832/134408591-b9faa51c-3b35-4cf2-992d-51dcd07c5cb5.png)
 
-Checkbox ``Don't throw Error on an Empty Result`` allows to emit an empty response, otherwise you will get an error on empty response.
+Dropdown **Emit Behaviour** contains following possible options:
+ * Fetch all - a single message with an array `results` will be emitted
```
Comment (A3a3e1) [inline README.md:208]:
```suggestion
 * Fetch all - a single message with an array `results` containing all the objects will be emitted
```

File: README.md [hunk 2]
```diff
@@ -201,9 +202,14 @@ Following types are supported:
...
+ * Emit Individually - multiple messages (one message per one row) will be emitted
+ * Expect Single - a single message with one result row will be emitted. If it is returned more than one result by query the error will be thrown. A boolean input "Allow Zero Results" (defaults to `false`) appears at input metadata. If `false` - error will be thrown, else - the empty object will be emitted.
```
Comment (A3a3e1) [inline README.md:210]:
```suggestion
 * Expect Single - a single message with one result row will be emitted. If more than one result is emitted the error will be thrown. A boolean input "Allow Zero Results" (defaults to `false`) appears at input metadata. If `false` - error will be thrown, else - the empty object will be emitted.
```

File: src/main/java/io/elastic/jdbc/actions/NewSelectAction.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    String sqlQuery = configuration.getString("sqlQuery");
+    String emitBehaviour = "emitIndividually";
+
+    try {
+      emitBehaviour = configuration.getString(EMIT_BEHAVIOUR);
+    } catch (NullPointerException e) {
+      LOGGER.info("No Emit behavior is specified, the default value Emit Individually will be used");
+    }
+
+    Utils.columnTypes = Utils.getVariableTypes(sqlQuery);
```
Comment (A3a3e1) [inline src/main/java/io/elastic/jdbc/actions/NewSelectAction.java:46]: That smalls bad. Violation of open-closed SOLID principle

Comment (shulkaolka) [reply src/main/java/io/elastic/jdbc/actions/NewSelectAction.java:46]: just copy past from existing actions

File: README.md [hunk 4]
```diff
@@ -205,9 +175,9 @@ Following types are supported:
 ![image](https://user-images.githubusercontent.com/16806832/134408591-b9faa51c-3b35-4cf2-992d-51dcd07c5cb5.png)
 
 Dropdown **Emit Behaviour** contains following possible options:
- * Fetch all - a single message with an array `results` will be emitted
+ * Fetch all - a single message with an array `results` containing all the objects (rows) will be emitted
  * Emit Individually - multiple messages (one message per one row) will be emitted
- * Expect Single - a single message with one result row will be emitted. If it is returned more than one result by query the error will be thrown. A boolean input "Allow Zero Results" (defaults to `false`) appears at input metadata. If `false` - error will be thrown, else - the empty object will be emitted.
+ * Expect Single - a single message with one result row will be emitted. If more than one result is emitted the error will be thrown. A boolean input "Allow Zero Results" (defaults to `false`) appears at input metadata. If `false` - error will be thrown, else - the empty object will be emitted.
```
Comment (A3a3e1) [inline README.md:180]:
```suggestion
 * Expect Single - a single message with one result row will be emitted. If more than one row is returned the error will be thrown. A boolean input "Allow Zero Results" (defaults to `false`) appears at input metadata. If `false` - error will be thrown, else - the empty object will be emitted.
```

**Review by A3a3e1 — APPROVED** [review]

**Pull request merged**